src/libstd/ffi/os_str.rs

```diff
@@ -188,6 +188,16 @@ impl OsString {
     /// in the given `OsString`.
     ///
     /// The collection may reserve more space to avoid frequent reallocations.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
```

Comment:
The API documentation conventions don't mention that we have to remove these annotations.

https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md